### ISSUE TITLE
Useage of CO2 Emission Layer in Analysis is strongly discouraged

### ISSUE THREAD

**Issue by goergen95** [mapme-initiative/mapme.forest#7]
Until some issues with the analysis of the CO2 Emission Layer have been resolved, its usage in analysis is currently discouraged. With integrating a new [dataset](https://data.globalforestwatch.org/datasets/gfw::forest-greenhouse-gas-emissions/about) some functions need to be adapted. Warnings are issued when using any functionality including the CO2 layer pointing towards this issue. This issue will contain updated information once available.

**Comment by Jo-Schie** [comment]
HI @goergen95. Can you tell us more specifically what the challenge is and which routines need to be implemented? From what I understand,  the Harris dataset shows gross emissions between 2000 and 2020. I am not sure how you would break that down to annual emission estimates...or is it that each pixel can show forest cover loss only once during that period and hence you could assign the pixel the emission value from the layer for the respective year?

**Comment by goergen95** [comment]
You are right, we could use that new data set actually like we used the old one. That workflow consists of detecting forest cover loss and then simply assign the associated CO2 value as a emission for that respective year. I was having the related [dataset](https://data.globalforestwatch.org/datasets/forest-carbon-removals/explore) in mind, that actually models carbon removed from the atmosphere by forests as a 20 year average which makes it necessary to divide by 20 if you consider only specific years.

Maybe I remove the warning then because the data can be used like before. But we can also consider to work in the forest carbon removal layer mentioned above so that we move away from a simple gross estimate but also consider forest regrowth?
